### ISSUE TITLE
Add aria-label to icon buttons

### ISSUE THREAD

**Issue by ianb** [mozilla-services/screenshots#2524]
Any buttons that don't have text should have an [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute).  These should also be localizable strings.  Known cases:

- [ ] Selection cancel
- [ ] Selection download

**Comment by ianb** [comment]
These have titles so I think they are okay
